docs/src/guide/customization/addons/redis.md

```diff
@@ -0,0 +1,109 @@
+# Redis
+
+The `Mollitia` [Redis](https://redis.io/) addon adds redis for some modules of every circuit. The list of modules coming with redis support are Ratelimit, SlidingCountBreaker and SlidingTimeBreaker.
+
+## Quick Start
+
+``` bash
+# Install mollitia
+npm install mollitia --save
+# Install Redis and the Redis addon
+npm install redis --save
+npm install @mollitia/redis --save
```

Comment:
I think it's ok to do that inline: `npm install @mollitia/redis redis --save`

Reply:
Indeed, updated